src/dispatch/event/service.py

```diff
@@ -184,6 +185,59 @@ def log_case_event(
     return event
 
 
+def log_signal_event(
+    db_session,
+    source: str,
+    description: str,
+    signal_id: int,
+    individual_id: int = None,
+    started_at: datetime = None,
+    ended_at: datetime = None,
+    details: dict = None,
+    dispatch_user_id: int = None,
```

Comment:
```suggestion
    individual_id: int | None = None,
    started_at: datetime | None = None,
    ended_at: datetime | None = None,
    details: dict | None = None,
    dispatch_user_id: int | None = None,
```

Reply:
We should do the same in the log incident and case functions.